mmv1/third_party/terraform/services/compute/resource_compute_instance_test.go.tmpl

```diff
@@ -1437,6 +1437,31 @@ func TestAccComputeInstance_scheduling(t *testing.T) {
 	})
 }
 
+func TestAccComputeInstance_schedulingTerminationTime(t *testing.T) {
+	t.Parallel()
+
+	var instance compute.Instance
+	var instanceName = fmt.Sprintf("tf-test-%s", acctest.RandString(t, 10))
+	now := time.Now().UTC()
+	terminationTime := now.Add(24 * time.Hour).Format(time.RFC3339)
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+		CheckDestroy:             testAccCheckComputeInstanceDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
could you add an update step here to ensure that changing the termination time works as expected?

Reply:
I added step where update is checked